frontend/app/(footer)/event-collection/[id]/page.tsx

```
@@ -0,0 +1,196 @@
+              {/* Tabs for Upcoming and Past Events */}
+              <div className="mt-12">
+                {/* Tab Navigation */}
+                <div className="flex items-center gap-1 mb-6 border-b border-gray-200">
+                  <button
+                    onClick={() => setActiveTab("upcoming")}
+                    className={`px-4 py-2 font-medium text-sm transition-colors ${
+                      activeTab === "upcoming"
+                        ? "text-core-text border-b-2 border-black"
+                        : "text-gray-500 hover:text-gray-700"
+                    }`}
+                  >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)